.changeset/breezy-crews-guess.md

```diff
@@ -0,0 +1,8 @@
+---
+"@spectrum-css/combobox": patch
+---
+
+Fast follow fixes for combobox
+
+- corrects style query for the `--system` reference to "legacy" in the combobox/themes/spectrum.css file
```

Comment:
container query 🤦‍♀️

Reply:
Good catch! I did a quick search for `--system: spectrum` and found one more instance of this in a `spectrum.css` file for **alert banner**. We should fix that too! Since it's a tiny related fix, I don't see a problem adding a commit and a separate changeset, but that could be a separate PR too.

Reply:
@jawinn I have another PR to fix alert banner!! 🎉